test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -125,7 +125,9 @@
     protected static final InetAddress nativeAddr;
     protected static final Set<InetAddressAndPort> remoteAddrs = new HashSet<>();
     private static final Map<ProtocolVersion, Cluster> clusters = new HashMap<>();
-    protected static final Map<ProtocolVersion, Session> sessions = new HashMap<>();
+    private static final Map<ProtocolVersion, Session> sessions = new HashMap<>();
```

Comment:
I was going to say that we could use `EnumMap<>(ProtocolVersion.class)` here and in `Clusters`, but I guess that the plan is to convert the keys in pairs of user and protocol version in a short while, so I guess this way is fine.